### ISSUE TITLE
Unsupported startup parameter in options: statement_timeout

### ISSUE THREAD

**Issue by mixmind** [pgbouncer/pgbouncer#907]
Hi, 
I found that changes were performed about options parsing but did not appear in the changelog.
Is it normal behavior that now I should specify all parameters that I want to use in options in ignore_startup_parameters?

For example:
`
PGOPTIONS='-c idle_in_transaction_session_timeout=22' psql -p 6432 -h localhost postgres
PGOPTIONS='-c session_timeout=22' psql -p 6432 -h localhost postgres
`

In pgbouncer 1.19.1 it worked, in 1.20.0 it started to work after changing the config to look like this:
`
ignore_startup_parameters = extra_float_digits, options , statement_timeout, idle_in_transaction_session_timeout
`

**Comment by JelteF** [comment]
Hmm, that's an interesting regression. By actually starting to support parsing the `options` parameter, we stopped supporting having `options` itself in `ignore_startup_parameters`. It should be fairly easy to fix that.

**Comment by mixmind** [comment]
> > If options is not provided in ignore_startup_parameters, and only other flags, it would not work too 😄
> 
> Could you share an example? Because with these settings:
> 
> ```
> ignore_startup_parameters = extra_float_digits, idle_in_transaction_session_timeout
> ```
> 
> this should be able to connect using 1.20.0 (there's even a test for this in our test suite):
> 
> ```
> PGOPTIONS='-c idle_in_transaction_session_timeout=22' psql -p 6432 -h localhost postgres
> ```

Yes, you are right, I forgot to change config properly, it actually works also if options is not in list

**Comment by JelteF** [comment]
Could you confirm that #908 fixes this issue for you?

**Comment by mixmind** [comment]
> Could you confirm that #908 fixes this issue for you?

Yes, now only `extra_float_digits, options` in `ignore_startup_parameters`  and it works

**Comment by joshuataylor** [comment]
I can also confirm the patch works for me as well.

**Comment by titasdutta** [comment]
Do we know when this will potentially be available?

**Comment by JelteF** [comment]
This is released now as 1.20.1 (rpm/deb packages might take a few days)
